geonode_mapstore_client/client/js/plugins/Share.jsx

```diff
@@ -125,7 +126,7 @@ function Share({
                     </Button>
                 </div>
                 <div className="gn-share-panel-body">
-                    <SharePageLink />
+                    <SharePageLink resourceType={resourceType} embedUrl={embedUrl} />
```

Comment:
I would prefer to have a component SharePageLink and pass the url and label instead to put everything inside one component

```js
<SharePageLink label={<Message msgId={} />} url={window.location.href} />
<SharePageLink label={<Message msgId={} /> url={embedUrl} />
```